.coderabbit.yaml

```
@@ -47,182 +49,343 @@
+    instructions: |
+      Review for Docker security best practices.
+      Ensure multi-stage build is used properly.
+      Check for minimal final image size and attack surface.
+      Verify proper handling of secrets and environment variables.
+
+# Pipeline Failure Remediation Configuration
+pipeline_remediation:
+  enabled: true
+  auto_suggest: true
+  auto_apply: false  # Start with manual review
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)